lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
```

Comment:
`schema` is not really `meta` it actually serves a purpose inside the rule itself. So I would leave it as is for now.